cypress.env.json

```diff
@@ -0,0 +1,7 @@
+{
+  "HOME_URL": "https://qa-react-conduit.herokuapp.com/",
+  "USER_EMAIL": "alan.reskin@wolox.com.ar",
+  "USER_NAME": "churry",
+  "USER_PASSWORD": "wolox1189",
+  "REGISTER_URL": "https://qa-react-conduit.herokuapp.com/register"
```

Comment:
use .env file and do not push it.

Reply:
and you only have to have the base URL, then you append the path you need in each page